app/services/users/safe_remote_profile_image_url.rb

```diff
@@ -0,0 +1,10 @@
+module Users
+  module SafeRemoteProfileImageUrl
+    # Basic check for nil and blank URLs, alongside likely incomplete URLs, such as just "image.jpg".
+    def self.call(url)
+      return Users::ProfileImageGenerator.call unless url.to_s.start_with?("https")
```

Comment:
Maybe it's just me, but I would find this easier to read if you reverse the logic:

```ruby
return url if url.to_s.start_with?("https")

Users::ProfileImageGenerator.call
```

Reply:
Changed!